app/views/clients/_form.html.haml

```diff
@@ -1,28 +1,36 @@
-= form_for(@client) do |f|
-  .form-inputs
+= semantic_form_for(@client) do |f|
+  = f.semantic_errors(*f.object.errors.keys)
```

Comment:
cool i hacked [some stuff to do this|https://github.com/apurvis/rader/blob/master/app/views/clients/new.html.haml#L3-L9] but this is way better